src/ENVPAK.CPP

```diff
@@ -20,7 +24,15 @@
 
 #include <envpak.h>	// declares functions in this file
 
-
+#if __has_include(<filesystem>)
```

Comment:
This filesystem passage seems to appear too many times.  However, I think the thing to do is merge, then look at encapsulating the filesystem interface.

Reply:
You are right. I reduce it to only appear in XIOPAK.H since all the other files call XIOPAK.H.